azurerm_function_app/main.tf

```diff
@@ -1,11 +1,17 @@
 # FUNCTIONS
 
+data "azurerm_storage_account" "azurerm_functionapp_storage_account" {
+  name                = "${var.azurerm_functionapp_storage_account_name}"
+  resource_group_name = "${var.resource_group_name}"
```

Comment:
I'd prefer to load the rg as data. Other parameters may be useful in other resources. Also, the rg name should come from the locals (copy from another variable/main file of other modules)

Reply:
Ok I fix it.